src/generate_jsdoc.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="../typings/shelljs/shelljs.d.ts"/>
+import shell = require("shelljs");
+
+function generate_doc(module: string, path: string) {
+    var moduleXml = module + ".xml"
+    var jsdoc = "jsdoc -r " + path + " -t templates/haruki -d console -q format=xml > " + moduleXml;
```

Comment:
`shelljs` doesn't provide such API. I'll use string interpolation. I didn't use it later just because when I was writting this exact code I didn't know that TypeScript has string interpolation. :)